homeassistant/components/dlib_face_identify/image_processing.py

```diff
@@ -34,7 +36,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 class DlibFaceIdentifyEntity(ImageProcessingFaceEntity):
     """Dlib Face API entity for identify."""
 
-    def __init__(self, camera_entity, faces, name=None):
+    def __init__(self, camera_entity, faces, name=None, tolerance=0.6):
```

Comment:
Remove the defaults for `name` and `tolerance` to improve maintainability as these parameters are explicitly passed when called in `setup_platform` and therefore are never be used.

Reply:
Ok, changes committed, thanks!